src/ApiPort/ConsoleProgressReporter.cs

```diff
@@ -77,6 +82,8 @@ public ConsoleProgressTask(string task, int? totalCount)
             {
                 await Task.Delay(1);
 
+#if RICH_CONSOLE
```

Comment:
Should we create a separate ConsoleProgressREporter? Will this still show up in the platforms that support it?